solvation_analysis/analysis.py

```diff
@@ -0,0 +1,127 @@
+# from abc import ABC
+from abc import ABC
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.analysis.rdf import InterRDF
+from MDAnalysis.analysis import distances
```

Comment:
so capped_distance actually residues in MDAnalysis.lib.distances, the reason why is working here is that you're importing distances which imports capped_distance, I'd just import directly from lib instead of implicitly importing this way.